HW2/P2/P2.py

```diff
@@ -22,30 +22,52 @@
     total = orig_counts.sum()
 
     # serial move
-    counts = orig_counts.copy()
-    with Timer() as t:
-        move_data_serial(counts, src, dest, 100)
-    assert counts.sum() == total, "Wrong total after move_data_serial"
-    print("Serial uncorrelated: {} seconds".format(t.interval))
+    times = []
+    for i in range(1):
```

Comment:
I think that maybe you could have achieved the same thing without the for loop here.